tests/integration/Newsletter/ViewInBrowser/ViewInBrowserRendererTest.php

```diff
@@ -165,31 +166,38 @@ public function testItReusesRenderedNewsletterBodyWhenQueueExists() {
       $this->diContainer->get(Shortcodes::class),
       $this->diContainer->get(Renderer::class)
     );
+    /** @var SendingQueuesRepository $sendingQueueRepository */
+    $sendingQueueRepository = $this->diContainer->get(SendingQueuesRepository::class);
```

Comment:
You get the SendingQueueRepository five times in different methods. It would be easier to store it one time in the constructor. What do you think?